CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.6.3 (2024-07-04)
+------------------
+* Feature: Flag to preserve original working directory when running notebooks to make local imports and relative paths work.
```

Comment:
Very helpful, thank you! Make sure to update the date when this does get released.

Reply:
Sure, release date bumped to 2024-07-11 hopefully we can get it released tomorrow
Will update if needed